yum/pom.xml

```diff
@@ -15,7 +15,7 @@
 
     <properties>
         <!-- note: the *bucket* is the root of the repo -->
-        <yumrepo.s3RepositoryPath>s3://nexus-private-artifacts/emodb/yum-repo</yumrepo.s3RepositoryPath>
+        <yumrepo.s3RepositoryPath>s3://bv-emodb-artifacts/emodb/yum-repo</yumrepo.s3RepositoryPath>
```

Comment:
Since Emo is public and this bucket is private, would it make more sense to just set this to a sample repo, such as `s3://example-bucket/path/to/yum-repo` and update Jenkins to set the correct value as a maven parameter?